dist/gmusic-theme.js

```diff
@@ -1008,155 +1013,155 @@ Converter.prototype.getValues = function(space) {
 
 module.exports = convert;
 },{"./conversions":3}],5:[function(require,module,exports){
-module.exports = {
```

Comment:
I mentioned that in the original post. Not sure - thought it might be an NPM version difference again, but I'm up to date this time.

What happens if you run `npm run build` on this branch?  Does it change it back?